CHANGELOG.md

```
@@ -1,52 +1,74 @@
 
+### Bug Fixes
 
+- **#135:** remove PAT usage ([#136](https://github.com/VilnaCRM-Org/website/issues/136)) ([77ffc1f](https://github.com/VilnaCRM-Org/website/commit/77ffc1fcf5a71094f9e542a1ed0ef9c8bc25dac6)), closes [#135](https://github.com/VilnaCRM-Org/website/issues/135) [#135](https://github.com/VilnaCRM-Org/website/issues/135) [#135](https://github.com/VilnaCRM-Org/website/issues/135) [#135](https://github.com/VilnaCRM-Org/website/issues/135) [#135](https://github.com/VilnaCRM-Org/website/issues/135) [#135](https://github.com/VilnaCRM-Org/website/issues/135) [#135](https://github.com/VilnaCRM-Org/website/issues/135)
 
 ## [1.2.1](https://github.com/VilnaCRM-Org/website/compare/v1.2.0...v1.2.1) (2025-02-14)
 
-
 ### Bug Fixes
 
-* **#121:** change token rotation logic ([#122](https://github.com/VilnaCRM-Org/website/issues/122)) ([81ba0b0](https://github.com/VilnaCRM-Org/website/commit/81ba0b03684f043dc9c3f090243d32e3f364f6f6)), closes [#121](https://github.com/VilnaCRM-Org/website/issues/121) [#121](https://github.com/VilnaCRM-Org/website/issues/121) [#121](https://github.com/VilnaCRM-Org/website/issues/121) [#121](https://github.com/VilnaCRM-Org/website/issues/121) [#121](https://github.com/VilnaCRM-Org/website/issues/121)
+- **#1:** infinite pnpm install ([#41](https://github.com/VilnaCRM-Org/website/issues/41)) ([23c5c18](https://github.com/VilnaCRM-Org/website/commit/23c5c18f0e7647fe2bd4b82a8c63fb131b69264d)), closes [#1](https://github.com/VilnaCRM-Org/website/issues/1) [#1](https://github.com/VilnaCRM-Org/website/issues/1)
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 100; Actual: 309

(MD013, line-length)